pkg/local_workflows/report_analytics_workflow_test.go

```diff
@@ -169,6 +168,52 @@ func Test_ReportAnalytics_ReportAnalyticsEntryPoint_validatesInput(t *testing.T)
 	_, err := reportAnalyticsEntrypoint(invocationContextMock, []workflow.Data{input})
 	require.Error(t, err)
 }
+
+func Test_ReportAnalytics_ReportAnalyticsEntryPoint_validatesInputJson(t *testing.T) {
+	// setup
+	logger := log.New(os.Stderr, "test", 0)
+	config := configuration.New()
+	orgId := "orgId"
+
+	config.Set(configuration.ORGANIZATION, orgId)
+	requestPayload := ""
+
+	input := workflow.NewData(workflow.NewTypeIdentifier(WORKFLOWID_REPORT_ANALYTICS, reportAnalyticsWorkflowName), "application/json", []byte(requestPayload))
+
+	// setup mocks
+	ctrl := gomock.NewController(t)
+	networkAccessMock := mocks.NewMockNetworkAccess(ctrl)
+	invocationContextMock := mocks.NewMockInvocationContext(ctrl)
+	testInitReportAnalyticsWorkflow(ctrl)
+
+	mockClient := newTestClient(func(req *http.Request) *http.Response {
```

Comment:
Question: Do we need a mock client for this test? I would assume that the error prevents any API call.

Reply:
Fixing it! Thank you 🙇